### ISSUE TITLE
multiple updates headers

### ISSUE THREAD

**Issue by danrabbit** [elementary/appcenter#1009]
Sometimes the updates view gets in a funky state where there are duplicates of the list headers. We should make sure to create those headers only once:

![Screenshot from 2019-05-28 10 05 21@2x](https://user-images.githubusercontent.com/7277719/58497178-3fc81000-8130-11e9-948d-d3b434d448df.png)
